README.md

```diff
@@ -1,2 +1,2 @@
 # typo3-crawler_widget
-A widget for the TYPO3 Crawler that would give TYPO3 Backend users an easier overview of potential problems with the Crawler
+A collection of widgets for the TYPO3 Crawler that give TYPO3 Backend users an easier overview of potential problems with the Crawler.
```

Comment:
I suggest "... that give TYPO3 backend users information about the status of Crawler." - which is more neutral.

Reply:
I changed this.